sdcm/cluster.py

```diff
@@ -3804,14 +3804,17 @@ def node_run_stress_bench(node, loader_idx, stress_cmd, node_list):
                                          (loader_idx, uuid.uuid4()))
             # Select first seed node to send the scylla-bench cmds
             ips = node_list[0].private_ip_address
-            bench_log = tempfile.NamedTemporaryFile(prefix='scylla-bench-', suffix='.log').name
-
-            result = node.remoter.run(cmd="/$HOME/go/bin/{name} -nodes {ips} |tee {log}".format(name=stress_cmd.strip(),
-                                                                                                ips=ips,
-                                                                                                log=bench_log),
-                                      timeout=timeout,
-                                      ignore_status=True,
-                                      log_file=log_file_name)
+
+            try:
+                result = node.remoter.run(
+                    cmd="/$HOME/go/bin/{name} -nodes {ips}".format(name=stress_cmd.strip(), ips=ips),
+                    timeout=timeout,
+                    log_file=log_file_name)
+            except Exception as exc:  # pylint: disable=broad-except
+                errors_str = DockerBasedStressThread.format_error(exc)
```

Comment:
1) this is not the correct place for `format_error` it should be in `stress_thread.py`
2) `format_error` -> `format_stress_cmd_error`